Docker/nvim/Dockerfile

```diff
@@ -41,9 +41,10 @@ USER admin
 
 ENV HOME=/home/admin \
     PYENV_ROOT=$HOME/.pyenv \
-    PATH=$HOME/.local/bin:/home/admin/.pyenv/shims:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/admin/.pyenv/bin:$PYENV_ROOT/plugins/pyenv-virtualenv/shims:$HOME/.pyenv/shims \
```

Comment:
The PATH variable don't have the next directory $HOME/.local/bin

Reply:
ok, Fixed